udaykalra/portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,140 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+:root {
+  --main-color: #fff;
+  --head-color: #000000;
+  --body-color: #121212;
+  --text-color: #e7e7e7;
+  --light-color: #333333;
+}
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+/*Dark theme*/
+html,
+body {
+  margin: 0;
+  height: 100%;
+  background-color: var(--body-color);
```

Comment:
The CSS style guide asks that declarations be alphabetically ordered:
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Also applies throughout this file.

Reply:
Alphabetized in latest commit.